lib/storage-client.js

```diff
@@ -103,7 +125,7 @@ class ChromedriverStorageClient {
    *
    * @param {string} driverKey - Driver version plus archive name
    * @param {string} notesUrl - The URL of chromedriver notes
-   * @param {Object} infoDict - The dictionary containing driver info.
+   * @param {any} infoDict - The dictionary containing driver info.
```

Comment:
maybe we could use https://stackoverflow.com/questions/42027864/is-there-any-way-to-target-the-plain-javascript-object-type-in-typescript

Reply:
we could, but it is `any` because it needs proper types, and those types come from some external system and I didn't want to bother to figure out what this should be.  I'll make a note of it.